ui/src/views/Tasks/TaskLog/index.jsx

```diff
@@ -37,12 +37,19 @@ export default class TaskLog extends Component {
   }
 
   render() {
-    const { classes, match, stream } = this.props;
+    const {
+      classes,
+      match,
+      stream,
+      data: { task },
+    } = this.props;
     const url = decodeURIComponent(match.params.logUrl);
     const run = this.getCurrentRun();
 
     return (
-      <Dashboard title="Log" disablePadding>
+      <Dashboard
+        title={task ? `Log ${task.metadata.name}` : 'Log'}
```

Comment:
```suggestion
        title={task ? `Log "${task.metadata.name}"` : 'Log'}
```

Reply:
@helfi92 Are the double quotes supposed to be displayed around the log name? 
If so, they are not showing up as quotes and other punctuation is being stripped from the displayed title, but I'm not sure where the code that's doing that is.

Reply:
>  Are the double quotes supposed to be displayed around the log name

Yeah. Why are they being stripped?

Reply:
The following works for me:

```diff
diff --git a/ui/src/views/Tasks/TaskLog/index.jsx b/ui/src/views/Tasks/TaskLog/index.jsx
index 3b773e4b6..9f0c3a373 100644
--- a/ui/src/views/Tasks/TaskLog/index.jsx
+++ b/ui/src/views/Tasks/TaskLog/index.jsx
@@ -48,7 +48,8 @@ export default class TaskLog extends Component {
 
     return (
       <Dashboard
-        title={task ? `Log ${task.metadata.name}` : 'Log'}
+        title={task ? `Log "${task.metadata.name}"` : 'Log'}
+        disableTitleFormatting
         disablePadding>
         <Helmet state={run && run.state} />
         <Log
```

Reply:
Thanks! That works for me too. The updated code is ready for review.